model/src/misc/raids.rs

```diff
@@ -0,0 +1,46 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{BulkEndpoint, Endpoint, EndpointWithId};
+
+pub type RaidId = String;
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Raid {
+    pub id: String,
+    pub wings: Vec<Wing>,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Wing {
+    pub id: String,
+    pub events: Vec<Event>,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Event {
+    pub id: String,
+    #[serde(rename = "type")]
+    pub type_field: String,
```

Comment:
I used `_type` as a naming convention in this repository. It's not optimal, but I would like if this would stay consistent